index.Rmd

```diff
@@ -93,6 +93,38 @@ fig
 Row {data-height=450}
 -----------------------------------------------------------------------
 
+### R Translation Teams | <small> Contact Details </small>
+
+```{r}
+translation_teams %>%
+  reactable(
```

Comment:
@benubah I'm less familiar with options here, I won't really comment on the aesthetics 😄

Reply:
Yeah... i didn't want to mess a lot of things up at once in this PR. But we need to move all the code for all the tables away from `index.Rmd` and write them up as functions and just call them up here